test/PowerShellEditorServices.Test.Shared/TestUtilities/TestUtilities.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.PowerShell.EditorServices.Test.Shared
+{
+    /// <summary>
+    /// Convenience class to simplify cross-platform testing
+    /// </summary>
+    public static class TestUtilities
+    {
+        private static readonly char[] s_unixPathSeparators = new [] { '/' };
+
+        private static readonly char[] s_unixNewlines = new [] { '\n' };
+
+        /// <summary>
+        /// Takes a UNIX-style path and converts it to the path appropriate to the platform.
+        /// </summary>
+        /// <param name="unixPath">A forward-slash separated path.</param>
+        /// <returns>A path with directories separated by the appropriate separator.</returns>
+        public static string NormalizePath(string unixPath)
+        {
+            if (unixPath == null)
+            {
+                return unixPath;
+            }
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux) || RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
```

Comment:
Why not do the reverse check of `if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)`

makes for a shorter `if`